lib/ast/node.rb

```diff
@@ -188,18 +188,21 @@ def to_a
     end
 
     # Converts `self` to a pretty-printed s-expression.
+    # Using the Sexp module the output can be used as ruby code to
+    # generate the expressions (handy for tests)
     #
     # @param  [Integer] indent Base indentation level.
     # @return [String]
     def to_sexp(indent=0)
       indented = "  " * indent
-      sexp = "#{indented}(#{fancy_type}"
+      sexp = "#{indented}s(:#{@type}"
```

Comment:
Er. This is not at all what I described in https://github.com/whitequark/ast/issues/6#issuecomment-141552159. The convention in Ruby is for `inspect` to output something you can run to get the object back (hence it should print the `s(:...` version of code, and `to_s` to output a readable representation (which already happens in `to_sexp`). There was no need to change `to_sexp`...